src/global/models/message.ts

```diff
@@ -1,32 +1,44 @@
 import { MessageType } from '../enums/message-type.enum';
+import { getUniqueId } from '../helpers/get-unique-id';
+
+function getMessageTitle(messageType: MessageType): string {
+  switch (messageType) {
+    case MessageType.ERROR:
+      return 'Error';
+    case MessageType.WARNING:
+      return 'Warning';
+    case MessageType.SUCCESS:
+      return 'Success';
+    default:
+      return 'Info';
+  }
+}
```

Comment:
Duplikat kodu? Wewnątrz klasy mamy praktycznie to samo, możesz to wywołać this.getTitle